### ISSUE TITLE
NVDA does not announce 'Disabled/Unavailable' information on "from" and "to" controls while navigating using tab key.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15267]
### Steps to reproduce:

1. Open the codepen URL: https://jsfiddle.net/bx7jku65/1/ 
2. Turn on NVDA
3. TAB till "btn1".
4. Now navigate to time edit field using down arrow.
5. Observe whether NVDA is announcing the disabled state or not

### Actual behavior:
NVDA does not announce 'Disabled/Unavailable' information on "from" and "to" controls while navigating using tab key.

### Expected behavior:
NVDA should announce as 'Disabled/Unavailable' information for "from" and "to" controls which are disabled while navigating using tab key.

#### NVDA installed/portable/running from source:
Windows 11 Enterprise
#### NVDA version:
NVDA Version 2023.1
#### Windows version:
22621.1992

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes-NVDA Version 2023.1

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/1eac148e-b5a7-4ca9-b04f-113f8e981d9b

**Comment by seanbudd** [comment]
This appears to be an authoring error. Is there a reason why the parent div was given a combobox role?

**Comment by msftedad** [comment]
Hi @seanbudd, Is there any update on this issue?

**Comment by seanbudd** [comment]
@msftedad the issue has already been closed. Please don't spam our GitHub bumping issues.
